src/tzkt/tzkt_block_api.py

```diff
@@ -28,9 +28,10 @@ def get_current_cycle_and_level(self) -> Tuple[int, int]:
 
         return (current_cycle, current_level)
 
-    def get_revelation(self, pkh, verbose=False):
+    def get_revelation(self, pkh):
         account = self.api.get_account_by_address(pkh)
-        return bool(account["revealed"])
+        revealed = account.get("revealed")
```

Comment:
Doing .get here means that if revealed isn't there it's not just going to fall over.